.github/workflows/docker-release.yml

```diff
@@ -13,17 +13,20 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
       - name: Login to GHCR
-        uses: docker/login-action@v1
+        uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build Docker image
-        run: |
-          docker build -t ghcr.io/timberware/woodpecker:latest -f deployments/Dockerfile .
-
-      - name: Push image
+      - name: Build and push multi-platform image

```

Comment:
Is there an action for this?

Reply:
There's one above, I think? I'm not sure, my docker knowledge is held together by scotch tape:


``yml
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2
```

Reply:
yeah, I mean maybe there's a `build-and-push` action (not instead but besides the one you already have) that supports multiplatform builds.

Reply:
Added!